www/templates/login.html

```diff
@@ -4,7 +4,7 @@
       <div class="rc-logo" ng-include="'templates/logo.html'"></div>
 
       <button class="button button-block button-positive" id="scan-to-login" ng-click="openScanner()">
-        Scan to Sign In
+        {{ "scan" | translate }} {{ "toSignIn" | translate }}
```

Comment:
I see what you're doing here but this is problematic because in other languages, the sentence may be structured differently (e.g. the verb may come last). It goes against a programmer's best instincts, but you need to treat translated strings like images -- they are blobs of text that you can't really add together. So please change this and all others like it to be a single string.